src/components/dialogs/parameters/sensi/sensitivity-analysis-parameters.js

```diff
@@ -0,0 +1,433 @@
+                                        [NAME]: sensiInjection[FILTER_NAME],
+                                    };
+                                }),
+                            [CONTINGENCIES]: sensiInjectionsSet[
+                                CONTINGENCIES
+                            ].map((sensiInjection) => {
+                                return {
+                                    [ID]: sensiInjection[FILTER_ID],
+                                    [NAME]: sensiInjection[FILTER_NAME],
+                                };
+                            }),
+                        };
+                    }) ?? [],
+            });
+        },
+        [reset]
+    );
+
+    useEffect(() => {
+        if (sensitivityAnalysisParams) {
+            fromSensitivityAnalysisParamsDataToFormValues(
+                sensitivityAnalysisParams
+            );
+        }
+    }, [
+        fromSensitivityAnalysisParamsDataToFormValues,
+        sensitivityAnalysisParams,
+    ]);
+
+    const clear = useCallback(() => {
+        reset(emptyFormData);
+        resetSensitivityParametersAndProvider();
+        resetSensitivityAnalysisParameters();
+    }, [
+        emptyFormData,
+        reset,
+        resetSensitivityAnalysisParameters,
+        resetSensitivityParametersAndProvider,
+    ]);
+
+    return (
+        <FormProvider validationSchema={formSchema} {...formMethods}>
+            <Grid
+                container
+                spacing={1}
+                padding={1}
+                key="sensitivityAnalysisParameters"
+                //className={classes.scrollableGrid}
+            >
+                <DropDown
+                    value={provider}
+                    label="Provider"
+                    values={providers}
+                    callback={updateProviderCallback}
+                />
+                <LineSeparator />
+                <SensitivityAnalysisFields
+                    reset={reset}
+                    useSensitivityAnalysisParameters={
+                        useSensitivityAnalysisParameters
+                    }
+                />
+                <LineSeparator />
+                <SensiParametersSelector
+                    reset={reset}
+                    useSensitivityAnalysisParameters={
+                        useSensitivityAnalysisParameters
+                    }
+                />
+
+                <DialogActions>
+                    <Button onClick={clear}>
+                        <FormattedMessage id="resetToDefault" />
+                    </Button>
+                    <SubmitButton onClick={handleSubmit(onSubmit)}>
+                        <FormattedMessage id="validate" />
+                    </SubmitButton>
+                    <CloseButton
+                        hideParameters={hideParameters}
+                        className={classes.button}
+                    />
+                </DialogActions>
+            </Grid>
+        </FormProvider>
```

Comment:
the unifying of the visual of parameters in #1423 & #1520 isn't respected
```suggestion
        <FormProvider validationSchema={formSchema} {...formMethods}>
            <Grid
                container
                spacing={1}
                padding={1}
                key="sensitivityAnalysisParameters"
            >
                <Grid
                    container
                    spacing={1}
                    sx={{ padding: 0, paddingBottom: 1 }}
                >
                    <DropDown
                        value={provider}
                        label="Provider"
                        values={providers}
                        callback={updateProviderCallback}
                    />
                </Grid>
                <LineSeparator />
                <SensitivityAnalysisFields
                    reset={reset}
                    useSensitivityAnalysisParameters={
                        useSensitivityAnalysisParameters
                    }
                />
                <LineSeparator />
                <SensiParametersSelector
                    reset={reset}
                    useSensitivityAnalysisParameters={
                        useSensitivityAnalysisParameters
                    }
                />
            </Grid>
            <Grid
                container
                className={classes.controlItem + ' ' + classes.marginTopButton}
                maxWidth="md"
            >
                <LabelledButton
                    callback={clear}
                    label="resetToDefault"
                />
                <SubmitButton onClick={handleSubmit(onSubmit)}>
                    <FormattedMessage id="validate" />
                </SubmitButton>
                <CloseButton
                    hideParameters={hideParameters}
                    className={classes.button}
                />
            </Grid>
        </FormProvider>
```

Reply:
done